src/components/gcds-alert/gcds-alert.tsx

```diff
@@ -0,0 +1,108 @@
+import { Component, Element, Host, Prop, h } from '@stencil/core';
+import { assignLanguage } from '../../utils/utils';
+
+@Component({
+  tag: 'gcds-alert',
+  styleUrl: 'gcds-alert.css',
+  shadow: true,
+})
+export class GcdsAlert {
+  @Element() el: HTMLElement;
+
+  private lang: string;
+
+  /**
+   * Defines the alert heading.
+   */
+  @Prop() alertHeading!: string;
+
+  /**
+   * Defines alert role.
+   */
+  @Prop() alertRole?: 'destructive' | 'info' | 'success' | 'warning' = 'info';
+
+  /**
+   * Defines if the alert's close button is displayed or not.
+   */
+  @Prop() hideCloseBtn?: boolean = false;
+
+  /**
+   * Defines the max width of the alert content.
+   */
+  @Prop() maxContentWidth?: 'fluid' | 'lg' | 'md' | 'sm' | 'xs' = 'lg';
+
+  /**
+   * Defines if the alert's position is fixed.
+   */
+  @Prop() positionFixed?: boolean = true;
+
+  /**
+   * Callback when the close button is clicked.
+   */
+  @Prop() onDismiss: () => void;
```

Comment:
To avoid having build warning everywhere for when we add this functionality to other components, maybe we should switch the naming of these types of props to `{event}Handler` instead of `on{event}`, so `onDismiss` would become `dismissHandler`.

Reply:
@ethanWallace updated!